documentation/book/api/index.adoc

```diff
@@ -0,0 +1,3785 @@
+:toc: left
+:numbered:
+:toclevels: 4
+:source-highlighter: highlightjs
+:keywords: openapi, rest, Kafka Bridge API Reference
+:specDir: 
+:snippetDir: ./documentation/book/api/snippet/
+:generator-template: v1 2019-12-20
+:info-url: https://openapi-generator.tech
+:app-name: Kafka Bridge API Reference
+
+= Kafka Bridge API Reference
+team@openapitools.org
+0.1.0
+
+== Introduction
+The Kafka Bridge provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol. 
+
+
+// markup not found, no include::{specDir}intro.adoc[opts=optional]
```

Comment:
What does this mean?

Reply:
You can have some additional adoc to be included. We don't have this intro.adoc so the openapi-generator tool is just putting this comment as placeholder.